test/unit/spec/v1/VerifyPIV_spec.js

```diff
@@ -135,6 +137,49 @@ Expect.describe('PIV', function() {
           });
         });
     });
+    itp('makes post call with correct data when isCustomDomain is false and customDomain is set', function() {
+      const config = {
+        certAuthUrl: 'https://foo.com',
+        isCustomDomain: false,
```

Comment:
Do we need a test to validate that `isCustomDomain: true` is ignored unless `customDomain` value is specified

Reply:
`isCustomDomain` isn't ignored - it works both ways. i.e. you could specify either one and it should work with a single custom domain. However, with. multiple custom domains, the backend will redirect to the most recently created custom domain.